dom/dom.php

```diff
@@ -11,7 +11,7 @@
  * </p>
  * @return DOMElement|null|false The <b>DOMElement</b> node added or <b>FALSE</b> if any errors occur.
  */
-function dom_import_simplexml (object $node): ?DOMElement {}
+function dom_import_simplexml ($node) {}
```

Comment:
Could you please clarify why `object` type hint should be removed here?

Reply:
PhpStorm itself reported "Argument type does not match the declared" because the phpdoc is "SimpleXMLElement" != "object", but I will revert the change, because I am not sure what is correct here.